README.md

```diff
@@ -181,6 +181,25 @@ make benchmarks
 
 The results will be output in `./benchmarks/results.md`.
 
+## strictObjectIDCoercion flag
+
+In version 1.17.0, where the id is of string type, is being converted to ObjectID when the string length is 12 or 24 and has the format of an ObjectID i.e /^[0-9a-fA-F]{24}$/. To avoid this issue, the strictObjectIDCoercion flag should be set to true in the model-definiton file.
+
+model-definition.js
+
+```js
+{
+  "name": "myModelName",
+  "base": "PersistedModel",
+  "idInjection": false,
+  "options": {
+    "validateUpsert": true,
+    "strictObjectIDCoercion": true
+  },
+...
+}
+```
```

Comment:
Can we have an example boot script to go with it?

@loay You can use the same example as I used for this [issue](https://github.com/strongloop/loopback-connector-mongodb/issues/52#issuecomment-309285598)